portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,31 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 700px;
 }
 
-#greeting-container {
-  margin-top: 20px;
+.font {
+  font-family: Optima, Arial, Helvetica, sans-serif;
+}
+
+@media only screen and (max-width: 700px) {
```

Comment:
It might be better to set the `max-width` to 699 so it's clear which styles will apply at exactly 700px. Otherwise I think the last rules declared in the stylesheet will override earlier ones.

Reply:
Good point!